### ISSUE TITLE
No Contributing.md file for the project

### ISSUE THREAD

**Issue by petercr** [ladybug-podcast/ladybugpodcast#66]
I mean it's _not really_ an issue, I just figured it couldn't hurt to have a contributing file with guidelines. I mean GitHub is always blabbing about how important they are 🤷‍♂ 🤷‍♀ 

It might be a nice little project for someone to put together 😄 

Thanks so much your show is awesome! 😍

**Comment by designMoreWeb** [comment]
I agree with this point. Some references for really good contributing files are:

- The Atom [Editor](https://github.com/atom/atom/blob/master/CONTRIBUTING.md) 
- Ruby on [Rails](https://github.com/rails/rails/blob/master/CONTRIBUTING.md) 
-  Open [Government](https://github.com/opengovernment/opengovernment/blob/master/CONTRIBUTING.md)

I got these from github's own documentation

**Comment by petercr** [comment]
I guess I'll just go ahead and make a PR with a  contributing file, and see what they say... 🤔 
I was basically thinking the same thing, GitHub points out that you should add one to your repos.
Rather than have _somebody_ do _something_ that makes you **want** to put one in. 

Besides I won't be heart broken if it doesn't get merged 👍 😆
